### DISCUSSION TITLE
Vertical Alignment in `LineChart` for Closely Ranged Key Values

### DISCUSSION THREAD

**Discussion by limitless-dev** [recharts/recharts#3708] [Q&A] [answered]
I've encountered an alignment issue in the `LineChart` component when the data values fall within a closely ranged span (e.g., 100 to 300).

Here's a visual representation of the issue:
![2023-08-18 at 12 19 27@2x](https://github.com/recharts/recharts/assets/24194621/1d08255e-079c-4e19-9d62-9f73961fd3b1)

When I adjust a single month's value to fall outside this range, the chart aligns correctly:
![2023-08-18 at 12 26 07@2x](https://github.com/recharts/recharts/assets/24194621/34598060-411d-4a9f-aca8-7ecf6f556422)

Could you advise on how to do proper alignment even when data values are within a tight range?

```tsx
    <ResponsiveContainer width="100%" height="100%">
      <LineChart data={data} >
          <ReferenceLine y={average} />
        <Line
          type="monotone"
          strokeWidth={2}
          dataKey={dataKey}
         //...
        />
      </LineChart>
    </ResponsiveContainer>

```

**Comment by ckifer** [comment, marked as answer]
You can add a YAxis with a domain that restricts the min and max values of the chart. <YAxis domain={["dataMin", "dataMax"]} />

**Reply by limitless-dev** [reply]
Worked perfectly, thank you!
